src/lib/storage/index/partial_hash/partial_hash_index_impl.cpp

```diff
@@ -226,12 +226,15 @@ typename PartialHashIndexImpl<DataType>::Iterator PartialHashIndexImpl<DataType>
 
 template <typename DataType>
 size_t PartialHashIndexImpl<DataType>::memory_usage() const {
-  size_t bytes{0u};
-  bytes += (sizeof(std::unordered_set<ChunkID>) + sizeof(ChunkID) * _indexed_chunk_ids.size());
+  auto bytes = size_t{0u};
+
+  bytes += sizeof(_indexed_chunk_ids);  // NOLINT
```

Comment:
Adding a comment after NOLINT explaining what the linter would complain about might be helpful.